tests/test_wave_01.py

```diff
@@ -143,12 +145,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
+    assert updated_data["watched"][1] == HORROR_1
```

Comment:
The spec for the function `watch_movie` doesn't state how movies are added to the `watched` list, so it's possible that it could add it to the front of the list rather than the end.  This test requires that the movie be appended to the end of the list.  Changing this to an `in` test would make the test work for different types of implementations:
```suggestion
    assert HORROR_1 in updated_data["watched"]
```